src/app/component/startquiz/startquiz.component.ts

```diff
@@ -1,37 +1,34 @@
 import { Component, OnInit } from '@angular/core';
 import { ActivatedRoute, Params, Router } from '@angular/router';
-import { QuestionService } from 'src/app/service/question.service';
+
 import quizData from '../../../assets/json/data.json';
 
 @Component({
   selector: 'app-carddesign',
   templateUrl: './startquiz.component.html',
-  styleUrls: ['./startquiz.component.scss']
+  styleUrls: ['./startquiz.component.scss'],
 })
 export class StartquizComponent implements OnInit {
-  quizData:any = quizData;
-  rules: any;
-  selectedQuizType: any;
-  constructor(
-    private route:Router,
-    private activeRoute: ActivatedRoute,
-    public questionService: QuestionService
-  ){}
-
+  quizData = { ...quizData };
+  instruction: any;
+  selectedQuizType!: string;
```

Comment:
Please add space between here

Reply:
yes i added space between constructor and variables